vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Lifecycle.class/retirement.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: retirement
```

Comment:
Let's hold off adding the retirement stuff now until we get further with PR #1971.  This instance and the Lifecycle.class can be removed from this PR.